### PR TITLE
Update portfolio

### PR DESCRIPTION
**Pull request by pr3pony**

### CODE REVIEW

File: portfolio/src/main/webapp/index.html [hunk 1]
```diff
@@ -2,17 +2,40 @@
...
+
+      <div>
+        <i class="fa fa-github"></i><a href="https://github.com/prprprpony">My GitHub profile</a>
+        <i class="fa fa-code"></i><a href="https://codeforces.com/profile/pr3pony">My Codeforces profile</a>
+        <i class="fa fa-at"></i><a href="https://atcoder.jp/users/pr3pony">My AtCoder profile</a>
+        <i class="fa fa-envelope"></i><a href="mailto:3254tg89@gmail.com">My email</a>
+      </div>
+
+      <p>Click here to get a random quote from my favorite sitcom <a href="https://community-sitcom.fandom.com/wiki/Community_Wiki">Community</a>.</p>
       <button onclick="addRandomGreeting()">Hello</button>
```
Comment (shadyof) [inline portfolio/src/main/webapp/index.html]: The button here seems a bit unclear, maybe provide some instruction(s)?

File: portfolio/src/main/webapp/index.html [hunk 2]
```diff
@@ -2,17 +2,40 @@
 <html>
   <head>
     <meta charset="UTF-8">
-    <title>My Portfolio</title>
+    <title>Ji-Gui Zhang's Portfolio</title>
     <link rel="stylesheet" href="style.css">
     <script src="script.js"></script>
+    <!--<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.13.1/css/all.min.css"/>-->
```
Comment (shadyof) [inline portfolio/src/main/webapp/index.html]: This can be removed unless you want to comment something.

**Review by shadyof — COMMENTED** [review]
Basically Good! Leave some nitpicking comments.

**Comment by pr3pony** [comment]
please give me an approving review if you think it's ok

**Review by shadyof — APPROVED** [review]

**Pull request merged**